python/runtime/cudaq/kernels/py_chemistry.cpp

```diff
@@ -21,6 +22,36 @@ void bindChemistry(py::module &mod) {
           "kernel. This function takes as input the existing `cudaq.Kernel` to "
           "append to, pre-allocated qubits, list of parameters, number of "
           "electrons, and number of qubits as input, in that order.");
+
+  mod.def("num_hwe_parameters", &cudaq::num_hwe_parameters,
+          "For the given number of qubits and layers, return the required "
+          "number of hwe parameters.");
+  // use explicit overload as we have two overloaded signatures, i.e. with and
+  // without CNOT couplers
+  //   mod.def("hwe_cnot", &cudaq::hwe_cnot<kernel_builder<>>,
```

Comment:
Would you mind removing this comment of old-code?

Reply:
Removed commented out code line.